foyer/tests/test_graph.py

```diff
@@ -37,3 +37,13 @@ def test_lazy_cycle_finding():
         rule = SMARTSGraph(smarts_string='[C;{}]'.format(token))
         list(rule.find_matches(top))
         assert all([hasattr(a, 'cycles') for a in top.atoms()])
+
+
+def test_cycle_finding_multiple():
+    fullerene = pmd.load_file(get_fn('fullerene.pdb'), structure=True)
+    top, _ = generate_topology(fullerene)
+
+    _prepare_atoms(top, compute_cycles=True)
+    cycle_lengths = [list(map(len, atom.cycles)) for atom in top.atoms()]
+    expected = [5, 6, 6]
```

Comment:
I might be thinking about this incorrectly but isn't this an entire fullerene structure? How can there only be 3 cycles?

Reply:
Ah! Nice - LGTM